pdr-api/handlers/parks/_identifier/name/PUT/index.js

```diff
@@ -1,9 +1,13 @@
 const AWS = require("aws-sdk");
-const { TABLE_NAME, dynamodb, getOne, putItem } = require('/opt/dynamodb');
+const { TABLE_NAME, dynamodb, getOne } = require('/opt/dynamodb');
 const { DateTime } = require('luxon');
 const { sendResponse, logger } = require('/opt/base');
 const TIMEZONE = 'America/Vancouver';
 
+const ESTABLISHED_STATE = 'established';
+const HISTORICAL_STATE = 'historical';
+const REPEALED_STATE = 'repealed';
```

Comment:
These should be pulled in from the dynamo layer as they may be used elsewhere than the PUT.